src/components/research/Research.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const Research = () => {
+  return (
+    <div className="flex items-center justify-center font-seasons text-[3vw]">
```

Comment:
```suggestion
    <div className="flex justify-center font-seasons text-[3vw]">
```
remove this it won't do anything